### ISSUE TITLE
In the finetune depth anything v2 to metric depth estimation task, why is the " - “ used in loss instead of the " + " sign?

### ISSUE THREAD

**Issue by booker-max** [DepthAnything/Depth-Anything-V2#217]
![image](https://github.com/user-attachments/assets/6f928f31-4be6-4417-9eec-f258b2a1573c)

**Comment by eirtech** [comment]
Hi, 
It is how the SI log loss defines in general. You might want to take a look at "Scale Invariant Log Loss", which is a popular loss function for depth estimation, for more detail and mathematical proof.
